bin/ember

```diff
@@ -5,13 +5,13 @@
 process.title = 'ember';
 
 // require('time-require');
-var resolve = require('resolve');
-var exit = require('exit');
+const resolve = require('resolve');
```

Comment:
`const` for `require` assignments is my personal preference. 
According to missing style guide requirements and `eslint` silence I make a conclusion that it's acceptable.